hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/AbstractAbfsIntegrationTest.java

```diff
@@ -532,4 +528,36 @@ protected long assertAbfsStatistics(AbfsStatistic statistic,
         (long) metricMap.get(statistic.getStatName()));
     return expectedValue;
   }
+
+  protected void assumeValidTestConfigPresent(final Configuration conf, final String key) {
+    String configuredValue = conf.get(key);
```

Comment:
it should also check conf.get(key.<accountUrl>, as done in `AbfsConfiguration.accountConf(name)`. All the configs are checked on both key and key.<accountUrl>. Dev can give anything.

Reply:
Makes sense.
Taken

Reply:
tip: if you set the default to "" then the line below can just check for the string not being empty

Reply:
Taken